### PR TITLE
Adding -dUseCropBox flag to Ghostscript extract tiff from pdf

### PR DESCRIPTION
**Pull request by nchiasson-dgi**
**ISLANDORA-2511**: (https://jira.lyrasis.org/browse/ISLANDORA-2511)

# What does this Pull Request do?

This pull request adds the -dUseCropbox flag to the extract tiff from pdf Ghostscript to use the Cropbox dataset of the PDF.

# How should this be tested?

Steps to test this issue:

* Collect a PDF with differing CropBox and MediaBox data. (Sample data PDF attached to JIRA issue)
* Ensure configuration selection available on /admin/islandora/solution_pack_config/paged_content
* Ingest a PDF book object with “Use Cropbox” not selected
* Ingest a PDF book object with “Use Cropbox” selected
* When the “Use Cropbox” selection is enabled, the derivative is built using the “Cropbox” dataset.

# Interested parties
@Islandora/7-x-1-x-committers
@dannylamb
@bondjimbond

### CODE REVIEW

**Review by IAMlKeno — APPROVED** [review]

**Comment by nhart** [comment]
Any @Islandora/7-x-1-x-committers able to take a look at this? Please? :)

**Comment by DiegoPino** [comment]
@nhart give me a few minutes loooooking

**Comment by DiegoPino** [comment]
@nhart All good. But (a small one): I feel the wording of the admin form input needs to be either more verbose or link to some place where the argument is explained.  Could be even https://www.ghostscript.com/doc/current/Use.htm or a copy of that setting. I see a lot of other boxes in Section "5. 1 Switches for PDF files" so would be good to explain why this was added. it also just affects PDFs so i would also change the 'media' to 'PDF' to be extra verbose. Won't make this a 'change request' more like a 'soft request' we all can agree on is needed.
Thanks!

**Comment by nhart** [comment]
@DiegoPino  @nchiasson-dgi  updated the description to be more verbose.

**Comment by DiegoPino** [comment]
@nhart i restarted a travis failed check. Once that one is green i will approve. Looks good to me and thanks for addressing my concerns

**Comment by DiegoPino** [comment]
@nhart @nchiasson-dgi restarting failed test again (PHP 5.5/Fedora 3.5.. maybe even time to ignore those.. just in case, all the rest is passing). If they still fail i will just merge and add a note somewhere/ issue. Pull looks perfectly fine to me. Approving in the meantime

**Review by DiegoPino — APPROVED** [review]
One Travis-CI check failing in the whole Matrix. Have restarted 2 times and still nothing. PHP 5.5 + Fedora 3.5.

**Comment by nhart** [comment]
@DiegoPino  can you push that merge button?

**Comment by DiegoPino** [comment]
🤦

**Comment by nhart** [comment]
Thank you, much appreciated!

**Pull request merged**
